apps/ensindexer/ponder.config.ts

```diff
@@ -47,7 +35,7 @@ const activePlugins = ALL_PLUGINS.filter((plugin) => config.plugins.includes(plu
 
 // combine each plugins' config into a MergedPonderConfig
 const ponderConfig = activePlugins.reduce(
-  (memo, plugin) => mergePonderConfigs(memo, plugin.config),
+  (memo, plugin) => mergePonderConfigs(memo, plugin.createPluginConfig(config)),
```

Comment:
Plugin config required using factory function in order to calculate the `PluginConfig` type and pass it as a param in to `ENSIndexerPlugin` type.